src/test/java/de/adesso/kicker/user/UserServiceTest.java

```diff
@@ -51,19 +51,28 @@ void setUp() {
 
     @Test
     void testGetAllUsers() {
+        // when
         ArrayList<User> allUsers = new ArrayList<>(userService.getAllUsers());
+
+        // then
         assertTrue(allUsers.contains(user));
```

Comment:
Where does user come from? I would recommend you to have it declared in the `//given` part of the test.